Dockerfile

```diff
@@ -16,4 +16,3 @@ COPY . .
 RUN /usr/bin/python -m pip install pip -U && /usr/bin/python -m pip install -r tests/requirements.txt && python -m pip install -r requirements.txt
 RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.11.3/bin/linux/amd64/kubectl \
```

Comment:
do we intend to use v1.11.3 of `kubectl` ?

Reply:
fixed